include_core/omrportsocktypes.h

```diff
@@ -62,6 +62,24 @@
 #include <sys/time.h>
 #endif /* defined(J9ZOS390) */
 
+/**
+ * To define struct pollfd for AIX.
+ */
+#if defined(OMR_OS_AIX)
+#include <sys/poll.h>
+#endif /* defined(J9ZOS390) */
+
+/**
+ * To define pollfd for all systems but Windows and ZOS.
+ */
+ #if !defined(OMR_OS_AIX) && !defined(OMR_OS_WINDOWS)
+ #include <poll.h>
+ #endif /* !defined(J9ZOS390) && !defined(OMR_OS_WINDOWS) */
```

Comment:
This comment, and the comment at the start of the block say zos, but the opening `#if` says aix